test/watcher.js

```diff
@@ -212,7 +212,7 @@ group('chokidar', (beforeEach, test, group) => {
 		t.ok(logger.reset.notCalled);
 		t.ok(logger.start.notCalled);
 		t.ok(api.run.calledOnce);
-		t.strictDeepEqual(api.run.firstCall.args, [files, {runOnlyExclusive: false}]);
+		t.strictDeepEqual(api.run.firstCall.args, [files, {runOnlyExclusive: false, updateSnapshots: false}]);
```

Comment:
I would extract `{runOnlyExclusive: false, updateSnapshots: false}` into a variable and use it in all these asserts. Would make it easier to add additional options in the future.